src/test/java/api/support/fakes/FakeStorageModule.java

```diff
@@ -51,6 +51,10 @@
 public class FakeStorageModule extends AbstractVerticle {
   private static final Logger log = LogManager.getLogger(MethodHandles.lookup().lookupClass());
   private static final Set<String> queries = Collections.synchronizedSet(new HashSet<>());
+  public static final String PATRON_ACTION_SESSION_STORAGE_PATH =
+    "/patron-action-session-storage/patron-action-sessions";
+  public static final String SCHEDULED_NOTICE_STORAGE_PATH =
+    "/scheduled-notice-storage/scheduled-notices";
```

Comment:
Do you think it's a good idea to hardcode the knowledge of specific endpoints into this class? Should't only specific instances of this class know that? I know that we don't have access to instances of `FakeStorageModule` from the tests, and I understand that we need to make these instances return errors in tests sometimes, but can we think of a more generic way to achieve that using static methods of `FakeStorageModule`? Something like:
```
FakeStorageModule.fail(HttpMethod method, String url, HttpStatus statusCode) {...}
```

Reply:
upated, thanks